sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/StringExpressionsSuite.scala

```diff
@@ -331,6 +331,16 @@ class StringExpressionsSuite extends SparkFunSuite with ExpressionEvalHelper {
     // scalastyle:on
   }
 
+  test("translate") {
+    checkEvaluation(StringTranslate("translate", "rnlt", "123"), "1a2s3ae")
```

Comment:
I guess it's due to there's an implicit ExpressionConversions for this as a shortcut.